x-pack/plugins/endpoint/server/routes/endpoints.ts

```diff
@@ -66,6 +66,10 @@ function mapToEndpointResultList(
   queryParams: Record<string, any>,
   searchResponse: SearchResponse<EndpointData>
 ): EndpointResultList {
+  let totalNumberOfEndpoints = 0;
```

Comment:
```js
  let totalNumberOfEndpoints = searchResponse?.aggregations?.total?.value || 0;
```

Would that work here?

Reply:
Yes, slipped me to use this shorthand. Thanks.